src/index.ts

```diff
@@ -120,6 +120,10 @@ export type {
 } from './handlers/gestureHandlerTypesCompat';
 
 export { default as Swipeable } from './components/Swipeable';
+export {
+  Swipeable as BetterSwipeable,
+  SwipeableController,
+} from './components/BetterSwipeable';
```

Comment:
@jakub-gonet @kmagiera 

Can we not add these additional components into the `./index` as UI components(both Swipeable and BetterSwipeable) are not directly related to handling gestures?

Metro is able to not include an unused component to the result bundle only if they are being imported like `import Swipeable from 'react-native-gesture-handler/Swipeable'`. This was always sad for me(and not just me) to see these components included in every app i deploy without using them increasing bundle size and as a result app startup time. As a part of the new 2.0 release can we move them out of `./index.ts`?

Reply:
We plan to extract Swipeable and Drawer to a different package. The new implementation will be using Gesture Handler and Reanimated 2 and we don't want to have a direct dependency on Reanimated.

Reply:
Thank You for this great news!

Reply:
> We plan to extract Swipeable and Drawer to a different package. The new implementation will be using Gesture Handler and Reanimated 2 and we don't want to have a direct dependency on Reanimated.

When will you extract Swipeable to a different package? Any plans?